main/inc/lib/webservices/Rest.php

```diff
@@ -1646,4 +1647,29 @@ public function createSessionFromModel($modelSessionId, $sessionName, $startDate
 
         return [$newSessionId];
     }
+
+    /**
+     * @param $sessionId
```

Comment:
Please try to always write one or two lines about what the function will do, even if it seems obvious from the function name. Over time, functions can change and become less true to their name. Having a clear English explanation here will help maintain the code over a long period of time.